resources/privacy-and-gdpr.md

```diff
@@ -6,53 +6,52 @@ Before implementing these controls, read Adobe's [GDPR documentation](https://ww
 
 When Adobe provides software and services to an enterprise, Adobe acts as a data processor for any personal data it processes and stores as part of providing these services. As a data processor, Adobe processes personal data in accordance with your company’s permission and instructions, as set out in your agreement with Adobe. As a data controller, you can use the Experience Platform SDKs to support GDPR retrieve and delete requests from your mobile apps.
 
-## Set and get privacy status
+## Set and get consent collect
```

Comment:
`Update and get collect consent preferences`
After renaming, update the references below as well